### DISCUSSION TITLE
Clerk Dashboard - broken js snippet?

### DISCUSSION THREAD

**Discussion by piotrekkr** [clerk/javascript#3244] [Support request - DO NOT OPEN HERE] [answered]
Hello. Just got into Dashboard and then `API Keys` section and there is this nice `Quick Copy` where I can select frameworks or libs to quickly start using Clerk. When selecting Javascript I think I get broken example:

![image](https://github.com/clerk/javascript/assets/503782/515c20d2-ffd8-4786-adeb-d2bebadfe378)

It has `https://` twice. Is this correct?

**Comment by nikospapcom** [comment, marked as answer]
hello @piotrekkr and thank you for reporting this issue.

It's not correct, we already deployed a fix about that. Thank you 👍
